tests/utils/types/guardian.type.ts

```diff
@@ -1,4 +1,5 @@
 interface Guardian {
```

Comment:
I think that `id` field is related a database field, not a `Guardian` attribute. 
Maybe we can use these types in `domain/models`, nowadays I'm working with these abstractions to `Pet` and `Specie` in other branchs without `id` field.